tests/pyoozie/test_tags.py

```diff
@@ -387,54 +380,63 @@ def parse_datetime(string):
     return datetime.datetime.strptime(string, '%Y-%m-%dT%H:%MZ')
 
 
-def test_coordinator(coordinator_xml, expected_coordinator_options):
-    actual = tags.CoordinatorApp(**expected_coordinator_options).xml()
-    assert tests.utils.xml_to_dict_unordered(coordinator_xml) == tests.utils.xml_to_dict_unordered(actual)
+def test_minimal_coordinator(minimal_coordinator_options):
+    actual_xml = tags.CoordinatorApp(**minimal_coordinator_options).xml(indent=True)
+    actual_dict = tests.utils.xml_to_dict_unordered(actual_xml)
 
+    expected_xml = None
+    with open('tests/data/minimal_coordinator.xml', 'r') as fh:
+        expected_xml = fh.read()
+    expected_dict = tests.utils.xml_to_dict_unordered(expected_xml)
+
+    assert actual_dict == expected_dict
+
+
+def test_full_coordinator(minimal_coordinator_options):
+    full_coordinator_options = minimal_coordinator_options
+    full_coordinator_options.update({
+        'end': parse_datetime('2115-01-01T10:56Z'),
+        'concurrency': 1,
+        'throttle': '${throttle}',
+        'timeout': 10,
+        'execution_order': tags.EXEC_LAST_ONLY,
+        'parameters': {
+            'throttle': 1,
+        },
+        'workflow_configuration': {
+            'mapred.job.queue.name': 'production',
+        },
+    })
 
-def test_coordinator_end_default(coordinator_xml, expected_coordinator_options):
-    del expected_coordinator_options['end']
-    actual = tags.CoordinatorApp(**expected_coordinator_options).xml()
-    assert tests.utils.xml_to_dict_unordered(coordinator_xml) == tests.utils.xml_to_dict_unordered(actual)
+    actual_xml = tags.CoordinatorApp(**minimal_coordinator_options).xml(indent=True)
+    actual_dict = tests.utils.xml_to_dict_unordered(actual_xml)
 
+    expected_xml = None
+    with open('tests/data/full_coordinator.xml', 'r') as fh:
+        expected_xml = fh.read()
+    expected_dict = tests.utils.xml_to_dict_unordered(expected_xml)
 
-def test_coordinator_with_controls_and_more(coordinator_xml_with_controls, expected_coordinator_options):
```

Comment:
Is the `coordinator_xml_with_controls` fixture still used, or can it be :fire:'d?